src/constants/index.js

```diff
@@ -37,6 +37,10 @@ export const PROVISION_SOURCE_CLONED_DISK = 'Cloned Disk'; // PVC or upload imag
 
 export const PVC_ACCESSMODE_RWO = 'ReadWriteOnce';
 export const PVC_ACCESSMODE_RWM = 'ReadWriteMany';
+export const PVC_ACCESSMODE_DEFAULT = PVC_ACCESSMODE_RWO;
+export const PVC_VOLUMEMODE_FS = 'Filesystem';
+export const PVC_VOLUMEMODE_BLOCK = 'Block';
+export const PVC_VOLUMEMODE_DEFAULT = PVC_VOLUMEMODE_BLOCK;
```

Comment:
Hardcoded defaults should probably be RWO/File

Reply:
done